karate-e2e-tests/src/test/java/driver/16.feature

```diff
@@ -0,0 +1,14 @@
+Feature:
+
+  Background:
+  * driver serverUrl + '/16'
```

Comment:
please have all the XPath related tests in one feature please, just reuse 15.feature is fine, add more HTML to 15.html if needed